build.sbt

```diff
@@ -128,12 +130,23 @@ lazy val root = Project("ctd-omega-editorial-frontend", file("."))
       MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
     ),
     headerLicense := Some(HeaderLicense.MIT("2022", "The National Archives")),
+    resolvers += Resolver.githubPackages("rwalpole"),
     libraryDependencies ++= Seq(
       guice,
-      "org.webjars.npm"         % "govuk-frontend"     % "4.3.1",
-      "uk.gov.hmrc"            %% "play-frontend-hmrc" % "6.2.0-play-28",
-      "org.scalatestplus.play" %% "scalatestplus-play" % "5.0.0"  % Test,
-      "org.jsoup"               % "jsoup"              % "1.15.3" % Test
+      "dev.fpinbo"             %% "jms4s-simple-queue-service" % "0.0.1-53518bb-SNAPSHOT",
```

Comment:
Is this the correct version, given that we have the GitHub artefact repo?

Reply:
This is the version that is in the github artefact repo. It's still a snapshot version, I'm not sure if we should release a proper numbered version here. @rwalpole what do you think?

Reply:
@felixgb this is fine for now. This is a topic to be discussed with @adamretter as ideally we don't want this in my GitHub account at all and we want the released artifact to go to Maven Central.